src/main/scala/nl.knaw.dans.easy.springfield/Command.scala

```diff
@@ -58,10 +59,10 @@ object Command extends App
   val result: Try[FeedBackMessage] = opts.subcommand match {
     case Some(cmd @ opts.listPresentations) =>
       debug("Calling list-presentations")
-      getPresentationList(cmd.user()).map(_.mkString(", "))
+      getPresentationList(cmd.user(), Console.out).map(_.size + " presentations")
     case Some(cmd @ opts.listFiles) =>
       debug("Calling list-files")
-      getFileList(cmd.user()).map(_.mkString(", "))
+      getFileList(cmd.user(), Console.out).map(_.size + " files")
```

Comment:
This CSV output is separated from the OK, but gets combined with maven output of run.sh
@janvanmansum How to get around?
https://github.com/DANS-KNAW/easy-springfield/blob/4d8f393810b6408416c5926f779813fc0fff8c57/src/main/scala/nl.knaw.dans.easy.springfield/Command.scala#L205-L206

Reply:
As explained by @janvanmansum run-quiet.sh suppresses the maven output.